miniaudiopolicy.cpp

```diff
@@ -0,0 +1,39 @@
+/*
```

Comment:
It is just easier to start a separate process so it can be started on demand.

Reply:
Is that necessary? Or can it just be present for all devices now? A fake service doesn't take up much of anything.

Reply:
I still think that'll be heavier than one more class in minimedia. There are quite a few of them here already, and each is just a really slim object rather than a whole process. Separating the fake services into separate files is good, but I doubt there'll be conflicts in the main?

Reply:
> I still think that'll be heavier than one more class in minimedia. There are quite a few of them here already, and each is just a really slim object rather than a whole process. Separating the fake services into separate files is good, but I doubt there'll be conflicts in the main?

But all header's are used in main? I think a separate file for main is still better because of that all headers are included into main I remember there were conflicts earlier.

Reply:
Are there actual conflicts present now though? I'm not really comfortable with a separate binary/service running on all devices when it's only needed for a few. I think it should either be optional  again or combined with the rest of the services in minimedia.

Reply:
If it's not necessary to run a separate service for audiopolicy, then I think the fake service sources should be merged into the existing services sources too. They're handily all kept together to ease porting to a new Android version, and it's a shame to have to make a copy of another file that your device may not even need.

Reply:
Moved into the main service.